clang-tools-extra/docs/clang-tidy/Contributing.rst

```diff
@@ -664,6 +679,17 @@ compiled against the version of clang-tidy that will be loading the plugin.
 The plugins can use threads, TLS, or any other facilities available to in-tree
 code which is accessible from the external headers.
 
+Note that testing checks out of tree might involve getting ``llvm-lit`` from an 
+installed version of LLVM through the `Stand-alone Builds`_ section. Alternatively,
```

Comment:
I'm not sure that we want to explicitly state `Stand-alone Builds` as the way to go. It would probably be better to say 
> from an LLVM installation compiled from source.

And link to the getting started page, in general. On the other hand, that would make this less specific

Reply:
Changed it to the more general link.